modules/assim.sequential/R/sda.enkf_local.R

```diff
@@ -0,0 +1,523 @@
+#' @description This function provides complete support for the multi-core and multi-node computation on the general HPC system.
```

Comment:
Could  you clarify the origin of this script? Is this from one of @DongchenZ PRs?

Reply:
I think so. It's duplicated.

Reply:
@cami-webb So I suggest you do the following things:
1) Check if my current PR (https://github.com/PecanProject/pecan/pull/3544) resolves your issue from the past, cause you mentioned that you fixed some bugs along your development.
2) Testify if the SDA function 'sda.enkf_local` works on your end.